tests/data/spe1_st/tests/test_webviz_ert.py

```diff
@@ -30,8 +30,6 @@ def _verify_keys_in_menu(dash_duo_handle, plugin, keys, selector):
 
 @pytest.mark.spe1
 def test_webviz_parameter_comparison(get_ensemble_id, dash_duo):
```

Comment:
Looks good, but the actual call happens when the fixture `get_ensemble_id` is initialized, I think if the fixture is removed from the definition of the test, all the tests will from the file will fail

Reply:
Ok - i'll have a look :+1:

Reply:
Allright - i've refactored a bit to make things more clear. The problem here is that without the call to `get_ensemble_id` the fixture for starting the storage server was never launched.

Reply:
Utilizing autouse and session scope, forcing the server to startup before tests are ran. In addition i changed start_server to connect_or_start_server to avoid multiple calls to start_server - and then just return the url and token ( used in request_get fixture ) . This seems like a more clean approach to me.

Reply:
Using  `scope="session"` should call the fixture only once per session so technically there is no need for the replacement.